english/you-are-in-minecraft/gather_wood/tame_the_elephant/tame_the_elephant.md

```diff
@@ -1,2 +1,4 @@
 You know you always wanted to have a pet elephant in minecraft, so you can kill all those zombies with a single whip.
-You decide to tame it.
+You decide to tame it. But it needs a name! Do you name it:
+[George](george/george.md)? 
+[Esparanza](esparanza/esparanza.md)?
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.